action.yml

```diff
@@ -6,26 +6,39 @@ inputs:
     required: true
   step_status:
     description: "The status of the step to update"
-    default: 'not_started'
+    default: "not_started"
+  step_details_preview:
+    description: "Preview Text for .details"
```

Comment:
Should this be `details`?

Reply:
It is the text that shows up next to the details dropdown.

Reply:
I mean should the `.` be there?